tangelo/algorithms/variational/adapt_vqe_solver.py

```diff
@@ -63,10 +63,13 @@ class ADAPTSolver:
         n_electrons (int): Self-explanatory.
         optimizer (func): Optimization function for VQE minimization.
         backend_options (dict): Backend options for the underlying VQE object.
+        simulate_options (dict): Options applicable to get_expectation_value e.g. desired_meas_result
```

Comment:
I thought it'd be applicable to `simulate`. Which is used to compute the expectation value.